cmd/gateway/commands.go

```diff
@@ -315,6 +319,13 @@ func createStaticModeCommand() *cobra.Command {
 			"A Lease object with this name will be created in the same Namespace as the controller.",
 	)
 
+	cmd.Flags().BoolVar(
+		&disableProductTelemetry,
+		productTelemetryDisableFlag,
+		false,
+		"Disable the collection of product telemetry every 24 hours.",
```

Comment:
nit:
I don't think we need to provide "every 24 hours". 
https://github.com/nginxinc/nginx-gateway-fabric/issues/1319 will cover how often and what is collected.
If we later decide to change the interval, we will have less places to update the doc strings.